update pt-br norme to v4 (#430)
42School/norminette: pdf/pt_br.norm.tex

                arquivos ou diretórios) devem ter os
                nomes mais explícitos ou mais mnemônicos possíveis.

>             \item Usando variáveis globais que não são marcadas como constante e estática é proibido e é considerado um erro de norma, há menos que seja explicitamente permitido no projeto.

Reviewer: ```suggestion
            \item Usar variáveis globais é proibido e considerado um erro de norma, há menos que seja explicitamente permitido o uso no projeto.
```

                  Os operadores devem estar no início de uma linha.

                \item Estruturas de controle (if, while ...) devem ter chaves,
                a menos que contenham uma única linha.

> 	\item Funções seguidas de chaves, declaradores ou estruturas de controle devem ser precedidas e seguidas de uma nova linha.

Reviewer: ```suggestion
                \item Funções seguidas de chaves, declaradores ou estruturas de controle devem ser precedidas e seguidas de uma nova linha.
```

        \item O header da 42 deve conter informações atualizadas, incluindo o
          criador com login e email, a data de criação e a data da atualização

>           mais recente. Cada vez que o arquivo é salvo em disco, a informação deve ser atualizada automaticamente.

Reviewer: ```suggestion
          mais recente. Cada vez que o arquivo for salvo em disco, a informação deve ser atualizada automaticamente.
```

            \item Nomes de macro devem ser todos maiúsculos (uppercase).
            \item Você deve recuar caracteres que seguirem \#if, \#ifdef
                or \#ifndef.

> 	\item Instruções de pré-processamento são proibidas fora do escopo global.

Reviewer: ```suggestion
            \item Instruções de pré-processamento são proibidas fora do escopo global.
```

            \item Seus comentários devem estar em inglês. E eles devem ser
                úteis.

>             \item Um comentário não pode ser usado para justificar declarações ou  uma função 

Reviewer: ```suggestion
            \item Um comentário não pode ser usado para justificar declarações ou uma função 
```

				mal feita.

        \end{itemize}
	\warn{

> 		Uma declaração ou função mal feita normalmente vem com nomes não explícitos como f1, f2... para as funções e a, b, i, ... para as declarações.

Reviewer: ```suggestion
		Uma declaração ou função mal feita normalmente vem com nomes não inteligíveis como f1, f2, etc. para as funções e, a, b, i, etc. para declarações.
```

> 		Uma função que tem o único objetivo de evitar a norma, sem um único propósito lógico, também é considerado uma função mal feita.

Reviewer: ```suggestion
		Uma função cujo objetivo é evitar ou burlar a norma, sem um único propósito lógico, também é considerado uma função mal feita.
```

> 		Por favor lembre se que é desejável ter funções limpas e legíveis que realizem uma clara e simples tarefa cada. Evite qualquer técnica de ofuscação do código como \emph{one-liner}..

Reviewer: ```suggestion
		Por favor, lembre-se de que é desejável ter funções limpas e legíveis que realizem uma clara e simples tarefa. Evite qualquer técnica de ofuscação de código como \emph{one-liner}.
```
